tests/fixtures/modifiers.js

```diff
@@ -103,15 +105,15 @@ const modifiersFixtures = [
 		'expected': '(?:[Aa][^])',
 		'expectedFlags': ''
 	},
-	{
+	!IS_NODE_6 && {
```

Comment:
These two tests are failing on node.js 6. However, starting from Node.js 8, the test is passing as expected. I assume this is a node.js 6 bug and it has been fixed in later node versions. So we can skip this test for node.js 6.

Minimum reproduction:

```js
// returns false on Node.js 6, true on Node.js 8 and above
/(?:[\u{10000}])k/u.test("\u{10000}k")
```